pkg/log/log.go

```diff
@@ -74,7 +75,7 @@ type FilteredLogger struct {
 var Log = DefaultLogger()
 
 func InitializeLogging(comp string) {
-	flag.StringVar(&defaultComponent, "component", comp, "Default component for logs")
```

Comment:
This change removes a cli argument. Was that intended?

Reply:
I did not see any usage of this argument, we just always use default argument that we send as `comp` to `InitializeLogging`, so I think it just redundant